Jenkinsfile

```diff
@@ -43,7 +43,25 @@ pipeline {
       }
     }
 
-    stage('Lint') {
+    stage('Lint Changed Files') {
+      when { changeRequest() }
+      steps {
+        script {
+          files_to_lint = ''
+          try {
+            files_to_lint = getGithubChangedFiles('vets-api', env.CHANGE_ID.toInteger(),
+                              change_types: ['modified', 'added']).join(' ')
+          } catch(IOException e) {
+            echo "WARNING: Unable to fetch changed PR files from Github!"
+            echo "${e}"           
+          }
+        }
```

Comment:
@omgitsbillryan might it make sense to fall back here to linting everything instead of failing? So you'd get only changed files if things go smooth and current behavior if it doesn't with the getGithubChangedFiles method

Reply:
Good catch. I didn't realize that I undid that functionality when breaking these up into 2 separate jenkins stages. I'll put it back in place 👍

Reply:
Whoops, I didn't realize that it already does fall back to linting all fails. On line 50: `files_to_lint = ''`, then if `getGithubChangedFiles()` raises, we catch and keep marching on to line 59.

Reply:
Is there some way the code could be reworked to make this the behavior more obvious?